docs2/site/docs/guides/dataloader.md

```diff
@@ -89,9 +89,34 @@ method of the returned `IDataLoaderResult<T>`. You can use a synchronous or asyn
 `IDataLoaderResult<T>` if you wish to chain dataloaders together. This may result in the field builder's Resolve delegate
 signature looking like `IDataLoaderResult<IDataLoaderResult<T>>`, which is correct and will be handled properly by the execution strategy.
 
+### GetOrAddBatchLoader vs. GetOrAddCollectionBatchLoader
+
+The decision about whether to use the `GetOrAddBatchLoader` or `GetOrAddCollectionBatchLoader` method should be based on whether each key can link to only one or to multiple values:
+
+- use `GetOrAddBatchLoader` for one-to-one relationships, i.e. when each key links to a single value; think for example of resolving the user who made an order: each order maps to one (and only one) user;
```

Comment:
This needs work.  It says "each order maps to one (and only one) user", but the next sentence below says "...[and] each user may map to (zero or) more orders".  Those are both correct, and so it's clear that this here is a many-to-one relationship, not a one-to-one relationship, with the `GetOrAddCollectionBatchLoader` being used for the opposite end of the same relationship.

Reply:
I think we are basically saying the same thing, just with different wordings 😄 

I see this as a one-to-one relationship from the the order's side (see also the comments above), thus the "each order". Of course, more than one order will likely map to a user, as a user is able to make multiple orders. Yet, I would not personally try to be extremely precise here. What is imho important for a user of the library is what they need to know when resolving a field via a data loader, not what the underlying logical model is.